modules/network/vpc/outputs.tf

```diff
@@ -17,49 +17,49 @@
 output "network_name" {
   description = "The name of the network created"
   value       = module.vpc.network_name
-  depends_on  = [module.firewall_rules, module.cloud_router]
+  depends_on  = [module.cloud_router]
```

Comment:
please add `module.vpc`. the point here is to ensure that the firewall rules are active prior to all other modules.

Reply:
Added. Thank you!